1_get_condor_data/src/get_data.R

```diff
@@ -24,6 +24,6 @@ get_data <- function(master_list, master_list_ind, ind_file){
     }
     index <- index + 1
   }
-
+  gd_put(master_list_ind, as_data_file(master_list_ind))
   gd_put(ind_file, as_data_file(ind_file))
```

Comment:
cool! nice to know `gd_put` can be used reasonably comfortably in this way. i've thought about adding a default for the second argument to gd_put, local_source, as `gd_put(remote_ind, local_source=as_data_file(remote_ind), ...)` ... I steered away from that previously because I think it should be common for `local_source` to need to be something else, but i see that it would be convenient here...hmm.